src/schema/linkml/bertron_schema.yaml

```diff
@@ -156,7 +156,7 @@ classes:
           - value: "100 meter"
         from_schema: http://w3id.org/mixs/terms
         multivalued: false
-        range: QuantityValue
+        range: NamedQuantityValue
```

Comment:
this means that the data structure can be

```yaml
  altitude:
    value: 3817
    unit: m
```

instead of having to specify the attribute name and ID, i.e.

```yaml
  altitude:
    attribute:
      label: altitude
      id: MIXS:0000094
    value: 3817
    unit: m
```
We may want to alter this so that all the attribute-values are in the same form but for now this is easier.